code/modules/power/lighting.dm

```diff
@@ -890,6 +890,8 @@
 
 /obj/item/light/proc/on_entered(datum/source, atom/movable/L)
 	SIGNAL_HANDLER
+	if(!istype(L, /mob/living))
+		return
 
 	if(istype(L) && has_gravity(loc))
```

Comment:
```suggestion
	if(istype(L, /mob/living) && has_gravity(loc))
```

Reply:
Yes, moving it to the same if statement as the gravity check is what I think would be good because then if anyone wanted other types to interact with moving onto a light that isn't mob/living they could do it in the same function.

Reply:
I will fix the commit and push when I get home